CMakeLists.txt

```diff
@@ -4,10 +4,11 @@
 cmake_minimum_required(VERSION 3.19)
 
 # Handle user build options.
-option(ENABLE_DOCS "Enable generation of doxygen-based documentation." OFF)
+option(ENABLE_FV3ATM_DOCS "Enable generation of doxygen-based documentation." OFF)
 
 # Determine whether or not to generate documentation.
-if(ENABLE_DOCS)
+if(ENABLE_FV3ATM_DOCS)
```

Comment:
Will documentation for multiple submodules at once ever be generated, or will we always generate docs within each repository?

Reply:
Great question that I had not even considered. Let us ponder how that might look.

But for immediate purposes, let's get this working independently. UPP already has an independent documentation build, so we need to not tangle with it when building fv3atm docs.